tools/json/dump-molecules-schema.json

```diff
@@ -0,0 +1,121 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://download.lammps.org/json/dump-molecules-schema.json",
+    "title": "JSON schema for LAMMPS 'dump' format in the 'molecules' style",
+    "description": "Version 0.1; last updated 2025-10-26",
+    "type": "object",
+    "required": ["application", "format", "style", "revision", "timesteps"],
+    "properties": {
+        "application": {
+            "type": "string",
+            "const": "LAMMPS"
+        },
+        "units": {
+            "enum": ["lj", "real", "metal", "si", "cgs", "electron", "micro", "nano" ]
+        },
+        "format": {
+            "type": "string",
+            "const": "dump"
+        },
+        "style": {
+            "type": "string",
+            "const": "molecules"
+        },
+        "revision": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 1
+        },
+        "title": {"type": "string"},
+        "schema": {"type": "string"},
+        "timesteps": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "required": ["timestep", "molecules"],
+                "properties": {
+                    "timestep": {
+                      "type": "integer",
+                      "minimum": 0
+                    },
+                    "molecules": {
```

Comment:
It may not be needed and have much meaning for ReaxFF output, but should the molecules have molecule IDs (for example when outputting a trajectory by molecule from a future dump style json)?

Reply:
At any rate, the molecule ID would be an optional field and thus can be omitted, if so desired or for situations where the information is not available.